.gitignore

```diff
@@ -1,7 +1,7 @@
 *
 !**/
 !*.*
-nimcache/
+**nimcache**
```

Comment:
This change doesn't relate to the purpose of this PR, please remove these changes and make them in a different branch then create a separate PR for them so that they can be discussed separately.